### PR TITLE
feat: document GCP PubSub resource (4.4)

### PR DESCRIPTION
**Pull request by thalesmg**

### CODE REVIEW

**Comment by qzhuyan** [comment]
LGTM

**Comment by thalesmg** [comment]
Added Chinese version.

Please help verify it, as it was translated with deepl.com :joy_cat:

**Review by ieQu1 — DISMISSED** [review]

File: zh_CN/rule/bridge-gcp-pubsub.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## 设置
+
+1. 创建一个[服务账户](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount)，
+   在你的GCP账户中确保服务账户有权限，可以向需要的主题发布消息。
+2. 为该账户创建一个服务账户密钥，并下载其为 JSON 格式下载。
+3. 创建一个 PubSub 主题（记住，服务账户必须有权限来发布到该主题）。
+
+## 创建 GCP PubSub 资源
+
+转到 [EMQX 仪表板](http://127.0.0.1:18083/#/resources), 选择左边菜单上的
```
Comment (xiangfangyang-tech) [inline zh_CN/rule/bridge-gcp-pubsub.md]: delete "选择左边菜单上的", it is repeated with following "在左边的菜单栏上选择".

File: zh_CN/rule/bridge-gcp-pubsub.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# 桥接数据到 GCP PubSub
+
+## 设置
+
+1. 创建一个[服务账户](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount)，
+   在你的GCP账户中确保服务账户有权限，可以向需要的主题发布消息。
+2. 为该账户创建一个服务账户密钥，并下载其为 JSON 格式下载。
```
Comment (xiangfangyang-tech) [inline zh_CN/rule/bridge-gcp-pubsub.md]: "并下载其为 JSON 格式下载" should be "并下载其为 JSON 格式文件"

File: zh_CN/rule/bridge-gcp-pubsub.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+3. 创建一个 PubSub 主题（记住，服务账户必须有权限来发布到该主题）。
+
+## 创建 GCP PubSub 资源
+
+转到 [EMQX 仪表板](http://127.0.0.1:18083/#/resources), 选择左边菜单上的
+在左边的菜单上选择“规则引擎”项目，然后选择“资源”。
+然后，点击“创建”。
+
+在对话框中，选择“GCP PubSub”类型，并点击“选择文件”。
+来选择和上传你要使用的服务账户的服务账户 JSON 文件。
```
Comment (xiangfangyang-tech) [inline zh_CN/rule/bridge-gcp-pubsub.md]: “来选择和上传你要使用的服务账户的服务账户 JSON 文件。” should be "选择上传前面下载的服务账户JSON 文件。"

File: zh_CN/rule/bridge-gcp-pubsub.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+帐户的 JSON 文件。点“确认”。
```
Comment (xiangfangyang-tech) [inline zh_CN/rule/bridge-gcp-pubsub.md]: 账户的JSON文件repeated with above. delete it。
点“确认” should be 点击“确认”

**Review by ieQu1 — DISMISSED** [review]

File: zh_CN/rule/bridge-gcp-pubsub.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+## 设置
+
+1. 创建一个[服务账户](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount)，
+   在你的GCP账户中确保服务账户有权限，可以向需要的主题发布消息。
+2. 为该账户创建一个服务账户密钥，并下载其为 JSON 格式文件。
+3. 创建一个 PubSub 主题（记住，服务账户必须有权限来发布到该主题）。
+
+## 创建 GCP PubSub 资源
+
+转到 [EMQX 仪表板](http://127.0.0.1:18083/#/resources),
```
Comment (zmstone) [inline zh_CN/rule/bridge-gcp-pubsub.md]:
```suggestion
转到 [EMQX 仪表板](http://127.0.0.1:18083/#/resources)，
```

File: zh_CN/rule/bridge-gcp-pubsub.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+在对话框中，选择“GCP PubSub”类型，并点击“选择文件”。
+选择上传前面下载的服务账户JSON 文件。
+点击“确认”。
+
+![创建一个 GCP PubSub 资源](./assets/gcp_pubsub_1.png)
+
+## 创建规则和动作
+
+转到 [EMQX仪表板](http://127.0.0.1:18083/#/resources), 选择左边菜单上的
+“规则引擎”项目，然后选择“规则”。 然后点击“创建”。
```
Comment (zmstone) [inline zh_CN/rule/bridge-gcp-pubsub.md:25]:
```suggestion
转到 [EMQX仪表板](http://127.0.0.1:18083/#/resources)，选择左边菜单上的
”规则引擎“ 项目，然后选择“规则”。 然后点击“创建”。
```

Comment (zmstone) [inline zh_CN/rule/bridge-gcp-pubsub.md]:
```suggestion
转到 [EMQX仪表板](http://127.0.0.1:18083/#/resources)，选择左边菜单上的
“规则引擎”项目，然后选择“规则”。 然后点击“创建”。
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**